src/energyplus/ForwardTranslator/ForwardTranslateEnergyManagementSystemActuator.cpp

```diff
@@ -65,113 +69,203 @@ namespace openstudio {
+      IdfObject idfObject = createRegisterAndNameIdfObject(openstudio::IddObjectType::EnergyManagementSystem_Actuator, modelObject);
+      const std::string spaceLoadNameForEMS = zoneOrSpaceName + " " + actuatedComponent_->nameString();
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentUniqueName, spaceLoadNameForEMS);
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentType, modelObject.actuatedComponentType());
+      idfObject.setString(EnergyManagementSystem_ActuatorFields::ActuatedComponentControlType, modelObject.actuatedComponentControlType());
+      return idfObject;
+    }
+
+    // ZoneName object does not exist so try and translate anyway
+
+    // the load references a space type, this means it will reference a ZoneList (if excludeSpaceTranslation = true) or a SpaceList in EnergyPlus
+    // in EnergyPlus, each load in a ZoneList is duplicated into each zone with a new name based on the zone, same for SpaceList
+    {
+      auto spaces = spaceType_->spaces();
+      if (spaces.empty()) {
+        LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                << "' and specified that it wanted to be attached to the SpaceType '" << spaceType_->nameString()
+                                << "' but the Space Type has no Spaces, so it will not be translated.");
+        return boost::none;
+      }
 
-        // Classic case, we just write it
+      std::string zoneOrSpaceName;
+
+      if (!m_forwardTranslatorOptions.excludeSpaceTranslation()) {
+        if (spaces.size() == 1) {
+          zoneOrSpaceName = spaces.front().nameString();
+          LOG(Warn, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString() << "' attached to the SpaceType '"
+                                 << spaceType_->nameString()
+                                 << "' but you have turned on ForwardTranslation's Space Feature. Falling back to using the attached Space '"
+                                 << zoneOrSpaceName << "' since there is only 1.");
+        } else {  // if (spaces.size() > 1) {
+          // TODO: do we grab the first space or just don't translate it?
+          std::sort(spaces.begin(), spaces.end(), WorkspaceObjectNameLess());
+          zoneOrSpaceName = spaces.front().nameString();
+          LOG(Error, "Actuator '" << modelObject.nameString() << "' references a SpaceLoad '" << load_->nameString()
+                                  << "' attached to the SpaceType '" << spaceType_->nameString()
+                                  << "' but you have turned on ForwardTranslation's Space Feature. The Space Type has multiple spaces "
+                                     "attached. Falling back to using the first Space '"
+                                  << zoneOrSpaceName << "'.");
+        }
```

Comment:
If we are translating to Spaces, we grab the first one. issue a warning if one only, an error if more than one.